frontend/src/feature-flags/greaterVersionThan.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+export function greaterVersionThan(versionA: string, versionB: string) {
```

Comment:
suggestion: there's no need to compare semver strings manually. Greater/less than operators work well:
```
'3.2.1' >= '3.2.1'   // true
'3.2.1' > '3.2.1'   // false
```

Reply:
Great point, will do